model/labels/labels_test.go

```diff
@@ -708,7 +708,8 @@ func TestScratchBuilder(t *testing.T) {
 
 func TestLabels_Hash(t *testing.T) {
 	lbls := FromStrings("foo", "bar", "baz", "qux")
-	require.Equal(t, lbls.Hash(), lbls.Hash())
+	hash1, hash2 := lbls.Hash(), lbls.Hash()
+	require.Equal(t, hash1, hash2)
```

Comment:
This seems to be provoked by lint, but I am struggling to imagine what it disliked.

Reply:
I think this only happen in the very latest version of golangci-lint. It says "useless-assert: asserting of the same variable (testifylint)". I would say that's a bug, because we are not really comparing the same variable, but the same function call, and that's totally valid as functions in Go might have side effects (oh yeah). Anyway, I thought the rewrite isn't really harmful, as one might argue it tells the reader what we are trying to test here.

Reply:
Thanks, that is very helpful. 
Now I agree with testifylint, it is quite a strange test.
Also I agree it's a bug in testifylint.